### ISSUE TITLE
Incompatible Datapack(s)

### ISSUE THREAD

**Issue by Coraline77447** [arollyson/FasterLadderClimbing#1]
Hi. I'm here to report an issue involving datapacks with this mod and why I'm reporting this about it's datapack(s) is cause I'm sick and tired of losing my worlds due to incomplete or missing data within the datapacks or outdated datapacks of some of the mods that I'm playing with which at some point I lose the world that I would be playing on and how I know this is cause when making a new world you can apply datapacks except for datapacks that mods adds and when I look in there I see some red ones that when I hover my mouse over them they say "Incompatible, made for an older version of Minecraft." When I would try to load into a modded world I get a message that says that the selected datapacks have errors and asks me to either enter safe mode (which I can't enter safe mode) or go to title screen and in the message it says that I can manually fix them which Idk how to fix incomplete, missing, outdated, or even broken datapacks to mods that has these errors!!! (Btw the mod works great but this is the only issue with it tho which is the mod's datapack(s) and I don't want to be playing my modded minecraft with mods that have Incompatible datapacks or else I'll lose my modded worlds cause the game will prevent me from loading the world ever again and I'm basically forced to delete the modded world and create a new one and I'm just so tired of doing that, I'm tired of losing my modded worlds that I've spent alot of time on only for it to be gone forever by this issue..... Also I hope that this mod's very  first issue report ain't too much to read.....)

**Comment by arollyson** [comment]
I'm not sure if this will fix the issue you're describing because your description is a little vague, but I updated the pack.mcmeta for the 1.16.4 version to 6. This may fix your issue, otherwise I don't think the problem is with this mod since it isn't touching data packs at all.

[Link to new version on curseforge](https://www.curseforge.com/minecraft/mc-mods/faster-ladder-climbing/files/3121858)

**Comment by Coraline77447** [comment]
This fix will fix the problem, and before you did that I may have done that same thing in the mods pack.meta at pack_format and I've done that with almost 25 other mods that I was playing with but there's some mods I can't change it's pack_format cause there's a line underneath called _comment that says what version for all of the owners mods must work for. If I were to change that while it was meant to be set at its current pack_format version then I could've done something bad to the mod or worse so I left the mods pack_format alone for the owner to do that themselves and an example of this is a mod called Artifacts that I can't change its pack_format which it's currently not set to 6, it's possibly set to 4 or 5 right now. Also what my report was originally about was about your mod's outdated datapack that houses your mods resources and I was trying to tell you to update it to version 6 but at the time of me posting my report I didn't have as much info about this till the mod owner of Blame mod told me what's going on with some of these mods that has this outdated datapack issue cause they simply forgot to update their mod's datapack or they didn't know that that's a thing or for any other reason that caused them not to update their datapacks. even if their mod doesn't touch datapacks at all the game doesn't care, if your mod technically uses datapacks for its resources then the game tracks it and if it's outdated then the game will give a warning to update it and in some cases if the problem isn't fixed depending on the mod's datapack that can cause damage to a modded world's datapack and can effect multiple modded worlds if you have the same mod that was used for multiple modded worlds, a prefect example of this is that I once made a modded world then I made copies of the same world and when one mod's datapack damaged the original worlds datapack and tried to load into one of the copies but can't load into them cause the copies uses the same stuff as the original did, so I had to dump all three worlds cause of that, but as of right now I've made a new world without any mods that has this issue and made a copy that'll have those mods installed so in case the copy gets destroyed I can simply uninstall those mods and play on the original that was made without any mods that has incompatible datapacks. (Btw sorry to have you read through this entire reply when it's too much too read...)
